Fix documentation about buildTarget (#2257)
scalameta/metals: docs/editors/vscode.md

      "type": "scala",
      "request": "launch",
      // configuration name visible for the user
      "name": "Launch Main",
      // full name of the class to run
      "mainClass": "com.example.Main",
      // optional arguments for the main class
      "args": [],
      // optional jvm properties to use
      "jvmOptions": []
    },
    // Test class configuration
    {
      "type": "scala",
      "request": "launch",
      // configuration name visible for the user
      "name": "Launch Test",
      // full name of the class to run
      "testClass": "com.example.Test"
    }
  ]
}
```

You can also add an optional build target name, which is needed in case there
are more than one class with the same name or when launching a class from
outside the project. Inside `"configurations":` add the key `buildTarget` with
your target name, e.g. `root`:

```json
      "buildTarget": "root"
```

> Build target name corresponds to your project name. For example in sbt for

Reviewer: ```suggestion
The build target name corresponds to your project name. For example in sbt for
```

Author: Fixed

`lazy val interfaces = project` the name of the build target will be

> `interfaces` for sources and `interfaces-test` for tests. To make sure what are
> the target names please run the command `Metals: Run Doctor`.

Reviewer: ```suggestion
`interfaces` for sources and `interfaces-test` for tests. To make sure you have the correct target names please run the command `Metals: Run Doctor`.
```

Author: Fixed!
